src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -133,6 +133,18 @@ public static function detectError($pdoOrStatement, SpanData $span)
         }
         $errorInfo = $pdoOrStatement->errorInfo();
         $span->meta[Tag::ERROR_MSG] = 'SQL error: ' . $errorCode . '. Driver error: ' . $errorInfo[1];
+
+        // Driver-specific error message will be in the rest of the array
+        // The spec suggests the error message to be at pos 2 only,
+        // but e.g. SQLServer doc states that the array can contain monre than 3 elements
+        if (count($errorInfo) > 2) {
+            $span->meta[Tag::ERROR_MSG] .= '. Driver-specific error message: ' ;
```

Comment:
This is not a blocker for me, but I am unsure `Tag::ERROR_MSG` is where we want to add this information, which to me seems to be pretty unpredictable. Should we add a tag like `error|err.details`? Again, very weakly opinionated on whether we should do it or not

Reply:
When we redefined the span attributes, we didn't include a specific one for such a need. 
The error message in some languages will contain the exception which will contain the same kind of data, so I don't think we should add a new one